media/EdgetpuCfgEditor/cfgeditor.css

```
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```

It seems like you need to change the copyright year for the newly created files.

```suggestion
 * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```

---

i fixed it. thanks :)